cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/TestCommand.java

```diff
@@ -196,6 +197,14 @@ public TestCommand() {
             "generation")
     private String graalVMBuildOptions;
 
+    @CommandLine.Option(names = {"--output", "-o"}, description = "Write the output to the given file. The provided " +
```

Comment:
```suggestion
    @CommandLine.Option(names = {"--output", "-o"}, description = "write the output to the given file. The provided " +
```

Reply:
Removed this flag as it is not needed (`--output`).